README.md

```diff
@@ -74,6 +78,12 @@ $ pip install -r requirements.txt
 $ make test
 ```
 
+### For SUSE distributions packages are available
+
+```
+$ zypper in aws-efs-utils
```

Comment:
I found an openSuse package [amazon-efs-utils](https://build.opensuse.org/package/show/home:glaubitz:branches:devel:languages:python:aws/amazon-efs-utils) but not one called `aws-efs-utils`. Can you link me to the package documentation with this name?

Reply:
"aws-efs-utils" seems to be the new name of the "amazon-efs-utils" package. You can find some information here: https://build.opensuse.org/package/show/Cloud:Tools/aws-efs-utils